pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/util/PackageSettings.java

```diff
@@ -91,14 +91,22 @@
     List<String> getSupportedLanguageVersions(T t);
 
     /**
-     * Determines if the package requires a build from source.
+     * Determine if the package always requires a build from source.
      * <p>
-     * Even if the binary artifact of the package is available,
-     * the package may need a rebuild because of custom build
-     * options or environment.
+     * Even if the binary artifact of the package is available in
+     * both global and local cache, the package may need a rebuild
```

Comment:
No need to explain the caching here or number of layers (global/local). Let's keep the previous description and just change the part after "because ..."